dev_guide/pod_autoscaling.adoc

```diff
@@ -212,21 +213,148 @@ $ oc create -f hpa.yaml
+
+You can use the status conditions set to determine
+whether or not the horizontal pod autoscaler is able to scale and whether or not it is currently restricted
+in any way.
+
+The following status conditions are set:
+
+* `AbleToScale` indicates whether the horizontal pod autoscaler is able to fetch and update scales, and
+whether any backoff conditions are preventing scaling.
+** A `True` condition indicates scaling is allowed.
+** A `False` condition indicates scaling is not allowed for the reason specified.
+* `ScalingActive` indicates whether the horizontal pod autoscaler is enabled (the replica count of the target is not zero) and
+is able to calculate desired scales.
+** A `True` condition indicates metrics is working properly.
+** A `False` condition generally indicates a problem with fetching metrics.
+* `ScalingLimited` indicates that autoscaling is not allowed because a maximum or minimum replica count was reached.
+** A `True` condition indicates that you need to raise or lower the minimum or maximum replica count in order to scale.
+** A `False` condition indicates that the requested scaling is allowed.
+
+To see the conditions affecting a horizontal pod autoscaler, use `oc describe hpa`. Conditions appear in the `status.conditions` field:
+
+----
+$ oc describe hpa cm-test
+Name:                           cm-test
+Namespace:                      prom
+Labels:                         <none>
+Annotations:                    <none>
+CreationTimestamp:              Fri, 16 Jun 2017 18:09:22 +0000
+Reference:                      ReplicationController/cm-test
+Metrics:                        ( current / target )
+  "http_requests" on pods:      66m / 500m
+Min replicas:                   1
+Max replicas:                   4
+ReplicationController pods:     1 current / 1 desired
+Conditions: <1>
+  Type                  Status  Reason                  Message
+  ----                  ------  ------                  -------
+  AbleToScale           True    ReadyForNewScale        the last scale time was sufficiently old as to warrant a new scale
+  ScalingActive         True    ValidMetricFound        the HPA was able to successfully calculate a replica count from pods metric http_requests
+  ScalingLimited        False   DesiredWithinRange      the desired replica count is within the acceptable range
+Events:
+----
+
+<1> The horizontal pod autoscaler status messages.
+
+The horizontal pod autoscaler status conditions are only available with the `v2alpha1` version of the
+autoscaling API by adding the following to your
+cluster's `master-config.yaml` file:
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    runtime-config:
+    - apis/autoscaling/v2alpha1=true
+----
+
+Restart the {product-title} services:
+
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+----
+endif::[]
```

Comment:
Maybe there's more to it, but I feel like the above (lines 306 -> 330) should be above the rest just under the "Viewing Horizontal Pod Autoscaler Status Conditions" heading.

Reply:
@bfallonf I moved the section up, just under "The following status conditions are set" list. Let folks see a bit of what they are getting by configuring the feature??

Reply:
@mburke5678 Sure, LGTM